web-local/package.json

```
@@ -58,6 +59,7 @@
     "should": "^13.2.3",
     "sinon": "^15.0.0",
     "supertest": "^6.2.4",
+    "svelte-awesome-color-picker": "^2.4.1",
```

Let's remove this dependency from the branch. I think the `<input type="color" />` should be sufficient
```suggestion
    "svelte-awesome-color-picker": "^2.4.1",
```